sitemapindex.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/siteindex.xsd"
+xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <sitemap>
+    <loc>https://www.mdanalysis.org/docs/sitemap.xml</loc>
+  </sitemap>
+  <sitemap>
+    <loc>https://www.mdanalysis.org/UserGuide/sitemap.xml</loc>
```

Comment:
I'm not sure about this one. This root-level sitemap lists root-level URLs, which just HTTP 301 redirect to the current release version (so https://www.mdanalysis.org/UserGuide/atomgroup.html redirects to https://userguide.mdanalysis.org/1.0.0/atomgroup.html).

There is also a https://userguide.mdanalysis.org/1.0.0/sitemap.xml that has the actual correct URLs for that version of the user guide.

[This seems to indicate that you should list the canonical URLs](http://www.thesempost.com/google-avoid-including-redirected-urls-sitemaps/) in a sitemap. In that case I should delete the root-level sitemap in the user guide (which also contains its own sitemap_index.xml that points to all the version-level sitemaps).

Reply:
I don't primarily think of SEO for the sitemaps but for the algolia site search – that's why the sitemaps are important. I want to avoid having to avoid the sitemapindex every time so if at all possible, I just want to include "the current released version".

Reply:
Sure -- maybe a good compromise would be to re-write UserGuide/sitemap.xml to have the right URLs with each update then (-> 1.0.0).

Reply:
Would this sitemap still point to UserGuide/sitemap.xml ?

Reply:
For right now I am following your suggestion and use the 1.0.0 path to make sure all our docs are releases.

We can then figure out how to not have to update this sitemaps file constantly.